_data/ja/toc.yml

```diff
@@ -171,6 +171,14 @@
       url: /ja/ch-ex/ex3
       not_numbered: true
 
+- title: 量子計算実験
+  url: /ch-labs/index
```

Comment:
@kifumi thank you for this recent update! it looks like the [index.md](https://github.com/kifumi/qiskit-textbook/blob/master-ja/content/ch-labs/index.md) was not included in [this translation](https://github.com/kifumi/qiskit-textbook/tree/master-ja/i18n/locales/ja/ch-labs). if possible can you translate it and include it also.